### PR TITLE
Add Symphony Testnet

### PR DESCRIPTION
**Pull request by dblackstone1**

### CODE REVIEW

**Review by editaahn — COMMENTED** [review]
Hi. Please check its README and make sure every validation test is successfully done.

File: images/symphony/chain.png
[the remark this answers is not in the document]
Comment (dblackstone1) [reply images/symphony/chain.png:1]: updated.  the directory has been renamed

File: cosmos/symphony-testnet.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  "chainId": "symphony-testnet-1",
+  "chainName": "Symphony",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/symphony/chain.png",
+  "rpc": "rpc.testnet.node2.symphonychain.org:26654",
+  "rest": "lcd.testnet.node2.symphonychain.org:1317",
+  "nodeProvider": {
+    "name": "Orchestra Labs",
+    "email": "dblackstone@orchestralabs.org",
+    "website": "orchestralabs.org"
+  },
```
[the remark this answers is not in the document]
Comment (dblackstone1) [reply cosmos/symphony-testnet.json]: added those with http (looking into getting certificates), and that passed verification on local so i pushed up those changes.  seems they're reflecting here.  i'll update once we have certificates

**Comment by dblackstone1** [comment]
> Hi. Please check its README and make sure every validation test is successfully done.

just verified on local after applying various changes to ensure compatibility.  validation now completes successfully

**Pull request merged**